machine_management/control_plane_machine_management/cpmso-getting-started.adoc

```diff
@@ -0,0 +1,42 @@
+:_content-type: ASSEMBLY
+[id="cpmso-getting-started"]
+= Getting started
```

Comment:
This section is commented out in the build and must be reviewed in a later PR that finishes this section.